hooks/post_gen_project.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 if [[ '{{ cookiecutter.init_repository }}' == 'y' ]]; then
-    git init --initial-branch=main
+    version_major=$(git --version | awk '{print $3}' | awk -F'.' '{print $1}')
+    version_minor=$(git --version | awk '{print $3}' | awk -F'.' '{print $2}')
+    version_major=$((version_major + 0))
+    version_minor=$((version_minor + 0))
+    if [[ $version_major -eq 2 && $version_minor -lt 28 ]]; then
+       git init
+       git co -b main
```

Comment:
Please do not use aliases

```suggestion
       git checkout -b main
```

Reply:
:man_facepalming: